modules/javafx.controls/src/main/java/javafx/scene/control/skin/TextAreaSkin.java

```diff
@@ -814,23 +828,20 @@ private char getCharacter(int index) {
     @Override
     protected int getInsertionPoint(double x, double y) {
         TextArea textArea = getSkinnable();
-        Text paragraphNode = getTextNode();
+        Text n = getTextNode();
```

Comment:
What's the reason for the name change `paragraphNode -> n`?

Reply:
The change made the following `if` statement easier to digest fit on a single line.

It's a local variable within a short function, I think it should be ok.